corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -82,10 +84,20 @@ hqDefine("data_dictionary/js/data_dictionary", [
                 );
                     groupObj.properties.push(propObj);
                 }
+                groupObj.properties.subscribe(self.changeSaveButton);
                 self.groups.push(groupObj);
             }
         };
 
+        self.fetchCaseProperties = function () {
+            if (self.groups().length === 0) {
```

Comment:
Nit: This can be refactored to simply be `if (!self.groups().length)` since a length of 0 will have a falsey value.